packages/@aws-cdk/cloudformation-include/test/test-templates/fn-sub-shadow-attribute.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": {
+          "Fn::Sub": [
+            "${AnotherBucket.DomainName}",
+            {
+               "AnotherBucket": "whatever"
+            }
+          ]
+        }
+      }
+    },
+    "AnotherBucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
```

Comment:
Again: this doesn't add anything to this test, so it shouldn't be here. `AnotherBucket` should look like this:

```json
    "AnotherBucket": {
      "Type": "AWS::S3::Bucket"
    }
```